Add blog post about persistent storage improvements in 0.12.0 (#78)
strimzi/strimzi.github.io: _posts/2019-03-01-resizing-persistent-volumes.md

> We hope that in one of the future versions of Strimzi we will add support for automatic resizing of persistent volumes through the operator.

Reviewer: ```suggestion
In future versions of Strimzi we hope to add support for automatic resizing of persistent volumes through the operator.
```

> But until then, you can still resize the storage manually using these simple steps and you don't have to delete the whole cluster and create a new one with bigger disks.

Reviewer: ```suggestion
Until then, you can still resize storage manually using a few simple steps. You don't have to delete the whole cluster and create a new one with larger disk capacity.
```
